client/js/app/modules/api_query_url.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
+var Qs = require('qs');
+var ExplorerUtils = require('../utils/ExplorerUtils');
+var TimeframeUtils = require('../utils/TimeframeUtils');
+
+function ApiQueryUrl(client, explorer) {
```

Comment:
What was the rationale for using a constructor here? It looks like there's only one function used (`getUrl`) and no need to keep object properties as state between function calls. You could export a single function here and not require the use of `new` when using the function in the `<QueryBuilder>` component.